Update description of VS Code extension for Marketplace (#1313)
microsoft/qsharp-compiler: src/VSCodeExtension/README.md

> # Microsoft Quantum Development Kit Preview for .NET6

Reviewer: is the right space .NET6 or .NET 6?

Reviewer: It looks like it should be .NET 6 https://dotnet.microsoft.com/en-us/download/dotnet/6.0

Author: I'll update it accordingly.

> # Microsoft Quantum Development Kit Preview for .NET6

Reviewer: ```suggestion
# Microsoft Quantum Development Kit Preview for .NET 6
```

This is a **preview version** of the Microsoft Quantum Development Kit extension for Visual Studio based on .NET6.0 components.

> We're actively working on supporting .NET6.0 in our VS Code extension. This support is still under development and this extension should be considered experimental.

Reviewer: ```suggestion
The Microsoft Quantum Development Kit extension supporting .NET 6.0 is under active development and should be considered experimental.
```

Reviewer: alternatively, "This extension is under active...."
Essentially the suggestion here is to remove possessive language.

> You can find here our currently supported extension [Microsoft Quantum Development Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).

Reviewer: ```suggestion
To use the currently supported extension, see [Microsoft Quantum Development Kit for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode).
```
